### ISSUE TITLE
Two sites that don't work

### ISSUE THREAD

**Issue by Nobabs27** [QupZilla/qupzilla#2209]
http://blankmediagames.com/TownOfSalem/
and
https://www.curse.com/home

![screenshot_2017-02-10_21-55-43](https://cloud.githubusercontent.com/assets/15079776/22850655/b9e38114-efdb-11e6-9df4-b8aa9be9fe12.png)
![screenshot_2017-02-10_21-55-29](https://cloud.githubusercontent.com/assets/15079776/22850656/b9ee6e76-efdb-11e6-8dad-b09bf04e0903.png)

**Comment by cranes-bill** [comment]
Regarding https://www.curse.com/, probably some _**Adblock** rule_ causing the problem. 
Regarding http://blankmediagames.com/TownOfSalem/, it just needs [**PepperFlash**](https://get.adobe.com/fr/flashplayer/?no_redirect) to be installed.

**Comment by Nobabs27** [comment]
Installed PepperFlash (from the AUR, I'm on Arch), still the same issue, sadly.
Going to see about AdBlock.
![screenshot_2017-02-11_17-11-25](https://cloud.githubusercontent.com/assets/15079776/22857905/31cf4784-f07d-11e6-8d68-059e9585efd8.png)

**Comment by Nobabs27** [comment]
Disabled AdBlock, still the same on Curse.

![screenshot_2017-02-11_17-17-57](https://cloud.githubusercontent.com/assets/15079776/22857934/14887870-f07e-11e6-90e8-415ebe1b8405.png)

**Comment by Gibbz** [comment]
Heres another site that doesnt work. Just displays the source html:
http://www.fabtab.org

**Comment by cranes-bill** [comment]
>Installed PepperFlash (from the AUR, I'm on Arch), still the same issue, sadly.


Probably in wrong place, you can try to specify it via Chromium option `--ppapi-flash-path="/full/path/to/libpepflashplayer.so"`, but that only works to me in Qt 5.6
I think in Qt 5.8, QtWebEngine needs the flash plugin to be installed in qt plugins directory

>Disabled AdBlock, still the same on Curse.


Did you try with clean profile? I can't reproduce it with 3 QupZilla versions, even with **mobile useragent** setting

>Heres another site that doesnt work. Just displays the source html:
http://www.fabtab.org


It does not works at all.

**Comment by Gibbz** [comment]
Yes http://fabtab.org doesnt work in qupzilla. Works in firefox, chrome
however.

**Comment by nowrep** [comment]
I have no issues with curse.com, fabtab.org doesn't work in chromium either and town of salem also doesn't detect flash here.

**Comment by Nobabs27** [comment]
> Did you try with clean profile? I can't reproduce it with 3 QupZilla versions, even with mobile useragent setting

How so?
Like, remove the .config/QupZilla then disable AdBlock?  If so, tried that, no change.

> Probably in wrong place, you can try to specify it

Ok, gonna give that a go.

**Comment by Nobabs27** [comment]
`$: qupzilla --ppapi-flash-path="/usr/lib/PepperFlash/libpepflashplayer.so"`
Result:
![screenshot_2017-02-12_15-33-28](https://cloud.githubusercontent.com/assets/15079776/22865770/a00f620e-f138-11e6-907d-efc66d79e850.png)
![screenshot_2017-02-12_15-34-17](https://cloud.githubusercontent.com/assets/15079776/22865774/ba7d4b6a-f138-11e6-813b-af6896d6684a.png)

The reason the window is half size is because it's split screen with another window.

**Comment by Gibbz** [comment]
weird that you dont have access to fabtab.

Heres what it should look like...
https://developers.google.com/speed/pagespeed/insights/?url=http%3A%2F%2Fwww.fabtab.org%2F&tab=desktop

**Comment by Gibbz** [comment]
this is what fabtab shows me...

btw github doesnt allow uploading of files/images through qupzilla either >.<

![fabtab](https://cloud.githubusercontent.com/assets/650506/22868656/27bcf036-f1e8-11e6-9584-4e6ff4a59d79.jpg)

**Comment by Gibbz** [comment]
ok ive fixed fabtab.org, however fonts are not rendering according to css.

**Comment by cranes-bill** [comment]
@Nobabs27 
>qupzilla --ppapi-flash-path="/usr/lib/PepperFlash/libpepflashplayer.so"
Result:...............................

Since Flash player works on the other sites, the problem is in http://blankmediagames.com/, the same as **fabtab.org**, where it works today.
So, you should tell of **blankmediagames.com** administrators about that.

**Comment by Nobabs27** [comment]
Ok, and what of Curse, that still does not work?

**Comment by Gibbz** [comment]
hangouts.google.com 
also fails to work

**Comment by cranes-bill** [comment]
> hangouts.google.com
> also fails to work

Due tu user agent, should to change it there!

**Comment by Nobabs27** [comment]
http://mail.yahoo.com crashes :\ 

Application version 2.1.2
QtWebEngine version 5.8.0

**Comment by iondissonance** [comment]
Plase, GDB backtrace or crashlog.

**Comment by Nobabs27** [comment]
Well QupZilla doesn't crash, it goes to a "Please Reload The Page" page.

**Comment by Nobabs27** [comment]
Reproduce:
1. Go to mail.yahoo.com
2. Login
3. Wait a few seconds then rip

**Comment by Nobabs27** [comment]
Actually it does not do it anymore, now that I opened in another web browser (it was some sort of promotional non-windowed popup).

**Comment by nowrep** [comment]
If the web process crashes - you get the page failed loading, please reload error - then it is a crash in QtWebEngine.

**Comment by Nobabs27** [comment]
I find this mildy insulting:
![screenshot_2017-03-29_18-45-21](https://cloud.githubusercontent.com/assets/15079776/24480329/bcad8354-14b1-11e7-95c6-17ad795538e3.png)

**Comment by cranes-bill** [comment]
I find that Yahoo is sucks! :smiley:

**Comment by nowrep** [comment]
You can change user-agent to make it look like you are using "modern" browser.

**Comment by Gibbz** [comment]
http://forums.overclockers.com.au
i found this forum (vBulliten) has massive issues working with qupzilla.
Clicking a link most of the time doesnt work, but if i middle click and open in a new tab, it usually works fine....

**Comment by Chiitoo** [comment]
Gibbz,

Is that with bd9a33d6f90a3c6229a56a0cee0816b92e66a937?  I had something very similar, if not identical happening elsewhere, before this fix.

Also, I can click around over at http://forums.overclockers.com.au just fine with:

Application version 2.1.99 (bd9a33d6f9)
QtWebEngine version 5.9.1

**Comment by Chiitoo** [comment]
Indeed the change would not be in the 2.1.2 release.

**Comment by Gibbz** [comment]
happen to know when we might see a 2.1.3 release with this fix?

**Comment by Chiitoo** [comment]
It looks like the next release will be 2.2.0 (https://github.com/QupZilla/qupzilla/issues/2302#issuecomment-292710579), which may be out at some point during this month (https://github.com/QupZilla/qupzilla/issues/2344#issuecomment-306521349).

**Comment by Adri-V** [comment]
Trying Qupzilla 2.1.2. 
Indeed, Yahoo mail is not working. Clicking "Try it later" brings me to the same page. 

![image](https://user-images.githubusercontent.com/24281230/30068135-1a8a59ba-925d-11e7-843c-0c2f45a6f589.png)

**Comment by Nobabs27** [comment]
I think you can get around it by just going to mail.yahoo.com or if that doesn't work "Try it later"

**Comment by cranes-bill** [comment]
@Adri-V, you must change user agent !!!

**Comment by Adri-V** [comment]
@Nobabs27 Going to mail.yahoo.com brings me to this page after logging in. Clicking "Try it later" does not work. 
@cranes-bill Could you please give some more details?
